src/components/PanelV2/__tests__/PanelV2.spec.js

```diff
@@ -3,38 +3,121 @@
  * @copyright IBM Security 2019
  */
 
-import { mount } from 'enzyme';
-import React, { Fragment } from 'react';
-
-import { label, onClick } from '../_mocks_';
+import { cleanup, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
 
 import { Button, PanelV2, PanelContent } from '../../..';
 
 describe('PanelV2', () => {
-  it('renders', () => {
-    const wrapper = mount(
+  afterEach(cleanup);
```

Comment:
I'm pretty sure you don't need to do clean up anymore. The package does this already.